ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -46,25 +47,28 @@ class ButtonDemoController: UITableViewController {
         case .textAndIcon,
              .textOnly,
              .iconOnly:
-            let cell = UITableViewCell()
+            let cell = tableView.dequeueReusableCell(withIdentifier: ButtonDemoController.cellReuseIdentifier, for: indexPath)
+            let subviews = cell.contentView.subviews
+            subviews.forEach { subview in
+                subview.removeFromSuperview()
```

Comment:
Do we do this in the other demo controllers? If no, should? What's the point of removing all the demo button cell's subviews?

Reply:
The difference is that before this change, we were never cacheing cells for reuse, which is a pretty inefficient way to use `UITableView`. Now that I switched it to a reuse mechanism, I'm following Apple's recommendation:

> The table view's delegate in tableView(_:cellForRowAt:) should _always_ reset all content when reusing a cell.

This is not a problem for `[MSF]TableViewCell`, which manages its own subviews during its `setup(title:subtitle:footer:customView:customAccessoryView:accessoryType:)` method.